src/test/Business/MetaQuestionController.test.js

```diff
@@ -0,0 +1,147 @@
+const {USER_TYPES} = require("../../main/Enums");
+const initSequelize = require("../../main/DAL/Sequelize");
+const {EMSError, MQ_PROCESS_ERROR_CODES} = require("../../main/EMSError");
+const Lecturer = require("../../main/business/UserManager/Lecturer");
+const {DEFAULT_PASSWORD} = require("../../main/business/UserManager/User");
+const MetaQuestionRepository = require("../../main/DAL/MetaQuestion/MetaQuestionRepository");
+const MetaQuestionControllerTest = require("../../main/business/MetaQuestions/MetaQuestionController");
```

Comment:
Why that name... LOL

Reply:
mistake, probably due to auto-complete suggestion which I didn't pay attention to

Reply:
anyway fixed